f3.go

```diff
@@ -122,8 +116,21 @@ func New(ctx context.Context, id gpbft.ActorID, manifest Manifest, ds datastore.
 		},
 	}
 
+	cs, err := certstore.OpenStore(ctx, ds)
+	if err != nil {
+		log.Warnf("opening CertStore, it might not have been created: %+v", err)
```

Comment:
We should distinguish between the two cases.

Reply:
Actually, I'd just _never_ open the cert store here. Instead:

1. On start, always wait until we've passed the bootstrap epoch.
2. Then open the store.